src/app.service.ts

```diff
@@ -141,19 +142,21 @@ export class AppService {
           'username, password, gender, birthdate, email all in string format required',
       });
     }
-    const user = await this.prismaService.user.findUnique({
-      where: {
-        username: username,
-        email: email,
-      },
-    });
+    // const user = await this.prismaService.user.findUnique({
+    //   where: {
+    //     username: username,
+    //     email: email,
+    //   },
+    // });
+    const user = null;
     if (user) {
```

Comment:
Use of this condition ?

Reply:
The whole file(app.service.ts) will be reconstructed due to changes in user schema after user routes will be recreated. The comments  and user = null are reason for that.